dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2326,7 +2325,6 @@
           "type": "string",
           "description": "Identifies the script of the input document.",
           "enum": [
-            "Unknown",
             "Latin"
```

Comment:
should the list contain all supported languages at the moment for documentation purposes?

Reply:
currently the model can detect latin script only. Will add more when we expand to other scripts. For input texts where script is not detected, this field is not returned.

Reply:
API board review - https://github.com/Azure/azure-rest-api-specs/issues/20051#issuecomment-1233099048

Reply:
I see, thanks for the clarification.